contracts/lendOrder.sol

```diff
@@ -0,0 +1,77 @@
+pragma solidity 0.4.15;
+
+contract LendOrder{
+    address public owner;
+    event NewOrder(uint id, uint _amount);
+    event NewUser(string _userName);
+
+    struct Order {
+        string owner;
+        uint amount;
+        uint32 time;
+        string typeOfOrder;
+    }
+
+    Order[] public orders;
+
+    mapping (address => uint) public balanceByAddr;
+    mapping (address => uint) public userIdByAddr;
+    mapping (address => string) public userNameByAddr;
+
+    function lendMoney(uint _amount) public onlyKnownUser() moreThanZero(_amount) {
+        uint id = orders.push(Order(userNameByAddr[msg.sender], _amount, uint32(now),"lend")) - 1;
+        balanceByAddr[msg.sender] =_safeAdd( balanceByAddr[msg.sender], _amount);
+        NewOrder(id, _amount);
+    }
+
+    function repayMoney(uint _amount, address _debtorAddress) public onlyOwner() moreThanZero(_amount) {
+        require(_debtorAddress > 0x0);
+        require(balanceByAddr[_debtorAddress] >= _amount);
+        uint id = orders.push(Order(userNameByAddr[_debtorAddress],_amount, uint32(now),"repay")) - 1;
+        balanceByAddr[_debtorAddress] = _safeSub( balanceByAddr[_debtorAddress], _amount);
+        NewOrder(id, _amount);
+    }
+
+    function showBalance() public onlyKnownUser() constant returns (uint) {
```

Comment:
what will happen if I'm not registered and try to look someones balance?

Reply:
I see
I have added private to balanceByAddr